jerry-core/jerry-port.h

```diff
@@ -142,6 +142,8 @@ double jerry_port_get_current_time (void);
 
 #ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
 
+#define JERRY_PORT_ENABLE_JOBQUEUE
```

Comment:
The marco is for main-unix.c, since I added the jobqueue related code in it.  Also the jerry-port-default-jobqueue.c used the macro. 

They are not in the jerry-core, so the profile config doesn't work on them